test/ffmpeg/ffmpeg_command_test.dart

```diff
@@ -113,6 +113,42 @@ void main() {
           ),
         );
       });
+
+      test("converts a group of images to a video", () {
+        final command = FfmpegCommand.simple(
+          inputs: [
+            // file%d.png represents images named like file0.png, file1.png, file2.png, etc.
```

Comment:
Instead of calling things `file` - how about we use names that reflect what a real file name for this purpose might look like. For example, I assume these files are screenshots?

Reply:
yah, they were. Will update the names.